src/app/routes/storage.ts

```diff
@@ -679,6 +743,13 @@ export default (router: Router, service: any) => {
     teamsAdapter,
     controller.getFolderContents.bind(controller),
   );
+  router.get(
+    '/storage/name-sorted/folder/:id/:idTeam?',
```

Comment:
We commented with @fabioespinosa to move the logic to a new endpoint because mobile is not ready to handle the pagination by name. It was originally meant to change `/storage/v2/folder/:id/:idTeam?`